### DISCUSSION TITLE
How should identity groups based on a shared ancestor be recorded?

### DISCUSSION THREAD

**Discussion by tychonievich** [FamilySearch/GEDCOM#461] [Q&A] [answered]
I had a version of the following question sent to me which I think might have broader interest than the specific person who sent it to me:

- How do we handle a group of people defined by their descending from a particular famous historical person?
- What about a subgroup based on which child or grandchild of that historical person it came from?
- What about if two such subgroups merge?

The message I received had various examples, but the person who shared it with me asked that I not share those details publicly. I can probably recreate anonymized examples from that upon request.

**Comment by dthaler** [comment]
I think the word "famous" here is a red herring.  Descent from a common ancestor is common for family associations and reunions, and sometimes each child of the common ancestor is referred to as a separate branch of the family, and so at reunions or family associations, identifying membership in the family association or the branch of the family at a reunion is common.  So I think the question here is how to track that in a GEDCOM file.

**Comment by Norwegian-Sardines** [comment]
> I think the word "famous" here is a red herring. Descent from a common ancestor is common for family associations and reunions, and sometimes each child of the common ancestor is referred to as a separate branch of the family, and so at reunions or family associations, identifying membership in the family association or the branch of the family at a reunion is common. So I think the question here is how to track that in a GEDCOM file.

I agree, "famous" is a red herring and subject to interpretation of "famous".  In the town my ancestors came from, my G-G-G-G-Grandfather was famous in town for building on to the church.  But you will not find him in any history book (other than the local history). However, identifying anyone that is in direct line from an individual (not necessarily a common ancestor to all at a reunion) is normally done using a numbering system like Henry or d'Aboville.  I've used a modified d'Aboville in my GEDCOM stored as a <USER_REFERENCE_NUMBER> with a REFN.TYPE indicating the referenced genealogy. 

```
1 REFN 1.5.3.1.4.1.1
2 TYPE <progenitor name> Descendants

```
The numbering system could be modified for better display in reports:

```
1 REFN John Doe.5.3.1.4.1.1
2 TYPE John Doe Descendants

```

**Comment by tychonievich** [comment]
Perhaps I should have said "historical" or "ancient" instead of "famous." The question arose in a context where there were centuries of unknown descendancy between the identity-defining person and the living identity-possessing people.

Consider a modern group of [Gershonite](https://en.wikipedia.org/wiki/Gershon) (I don't know if there are modern Gershonites, but the example translates to similar groups in many older cultures). They are defined by being descendants of Gershon, son of Levi, son of Israel. But there is a gap of more than a millennium in most genealogies: if you are a Gershonite you probably know that because your parents told you that your family descends from Gershon without a firm genealogy being available. Additionally, it is likely that any given Gershonite believes they have ancestry from all of the tribes of Israel, but that they "belong to" Gershon's branch where others do not because of how they descended (patrilineally, with possibly a few official adoptions or tribe switches along the way). They also likely believe they are Jewish, a group defined by the union of three ancestor-based identities (Judah, Benjamin, and half of Levi).

- Because of the gap in the genealogy, d'Aborville string won't work, nor will a full chain of `INDI`s linked with `FAMC`s.
- Because there are many Gershons (both historical people and places that could reasonably be said to have "Gershonites"), it's not immediately obvious how to use a string to unambiguously identify this meaning of "Gershonite" in a `NATI` or the like. A longer phrase there could work, of course, but the longer the phrase the more likely it is that different people from the same group will be recorded with different phrases.
- Because only some of Gershon's genetic descendants are socially considered to be "of Gershon," we need a bit stronger statement that "descendant of" -- maybe "inheriting descendant of" or the like instead?

**Comment by tychonievich** [comment]
The answer I gave was

1. Create an `INDI` for the historical individual
2. Add a unique identifier to that INDI, like [`EXID 79511` with `TYPE https://en.rodovid.org/wk/Person:`](https://en.rodovid.org/wk/Person:79511)
3. Add an `ASSO` to each person with the identity, pointing to that INDI, with `ROLE OTHER` and some defining `PHRASE` (maybe "-ite" or "descendant of" or "family progenitor"?), or perhaps defining a new extension ROLE.

... but that answer didn't fully satisfy me because of the awkward ROLE, and it doesn't extend to merged family groups like Judah+Benjamin+Levi. I'd welcome other proposals, either using standard tags or extensions.

**Comment by Norwegian-Sardines** [comment, marked as answer]
Because I don't fully understand what a [Gershonite](https://en.wikipedia.org/wiki/Gershon) is even after reading the linked page I can't associate it with anything that I do understand.

It seams to me like a tribe, clan or house.  If my interpretation was correct then of course I would use the `NATI` tag to indicate my inclusion in that group.  Being a member of a a tribe/clan/house would use the follow structure:

```
1 NATI <named group>
2 TYPE HOUSE | CLAN | TRIBE
```

Since the Levites are considered a "Tribe"

```
1 NATI Gershon son of Levi
2 TYPE TRIBE
```

But if "tribe" makes someone uncomfortable, then:

```
1 NATI Gershon
2 TYPE LEVITE
```

**Comment by Norwegian-Sardines** [comment]
As an extension to what I noted above, and using my previous calls for better "Typefication" in future releases of GEDCOM I would also make the following suggested changes to GEDCOM with discussion of potential enumset values:

```
n NATI <Text>                  {1:1} g7:NATI
+1 TYPE <Text>                 {0:1} g7:NATI-TYPE
+2 PHRASE <Text>               {0:1} g7:PHRASE
+1 <<INDIVIDUAL_EVENT_DETAIL>> {0:1}
```

<g7:enumset-NATI-TYPE>
NATIONALITY | HOUSE | TRIBE | CLAN | OTHER

Where these terms are not to be necessarily defined by GEDCOM, but rather defined by the user so the term matches their local use and understanding!

**Comment by dthaler** [comment]
Discussion in GEDCOM Steering Committee 9 MAY 2024:
* This might be worth adding to the FAQ.  Any reason to add example of NATI vs. ASSO in the spec?
* "Mayflower descent" might be another case of a family group within the US that some might use NATI for.
* There may also be some overlap in DNA with a desire to treat a (say) haplogroup as a NATI in existing GEDCOM files.
* What about membership in an organization, like Sons of the American Revolution or Daughters of the American Revolution in the US?  A living person might be a member of an SAR.  An ancestor might be recognized as being SAR/DAR eligible for descendants.
* One suggestion for potential future use might be a "descendant of" ROLE type for an ASSO to an ancestor.

**Comment by Norwegian-Sardines** [comment]
ASSO requires an individual to be added to the database and in some cases may be disconnected from the tree, in the example of Gershonite!  ASSO also requires an additional record query to retrieve the name of the person, and requires back pointers if a query is made to find all members associated with that individual, I.e. all descendants of a Mayflower passenger.  If a partial GEDCOM is exported the ASSO individual may not get exported.

An attribute using NATI may be faster to find in a query for all members of a clan or group.  An individual’s association with a group follows the individual record when the NATI is used just like most other attribute/events.  NATI is more likely provided in receiving applications over an ASSO relationship.

**Reply by Norwegian-Sardines** [reply]
Membership in an organization like SAR/DAR are grey but not directly genealogical in nature, in my opinion, should not be used in NATI.  The use of NATI in this way could get way out of context, (Business associations, fraternal groups) are extreme but could get include by some people looking for a place to put this data.

CORRECTION:  SAR/DAR are genealogical in nature, so they would be considered a NATI.  Sorry!!!!

**Reply by Norwegian-Sardines** [reply]
CORRECTION: SAR/DAR are genealogical in nature, so they would be considered a NATI. Sorry!!!!
